spec/lcm/integration/support/project_helper.rb

```diff
@@ -188,9 +216,7 @@ def create_tag_for_fact_n_dataset
 
     def ensure_user(login, domain)
       user = domain.users(login)
-      unless user
-        user = domain.add_user(login: login)
-      end
+      user ||= domain.add_user(login: login)
```

Comment:
info: Refers to 'domain' more than self (maybe move it to another class?) (FeatureEnvy) (reek)